### PR TITLE
Use OperationRef instead of DataloadRef and DataBackupRef

### PR DESCRIPTION
**Pull request by xliuqq**
### Ⅰ. Describe what this PR does
Use OperationRef instead of DataloadRef and DataBackupRef.

### Ⅱ. Does this pull request fix one issue?

None

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: pkg/dataload/constants.go [hunk 1]
```diff
@@ -16,6 +16,8 @@ limitations under the License.
 package dataload
 
 const (
+	DATALOAD_LOCK_NAME = "DataLoad"
+
```
Comment (cheyang) [inline pkg/dataload/constants.go:20]: Naming style for constants: Go follows the camel case naming convention for constants, so I suggest renaming DATALOAD_LOCK_NAME to DataLoadLockName.

File: pkg/databackup/constants.go [hunk 2]
```diff
@@ -17,6 +17,8 @@ limitations under the License.
 package databackup
 
 const (
+	DATABACKUP_LOCK_NAME = "DataBackup"
```
Comment (cheyang) [inline pkg/databackup/constants.go:20]: Naming style for constants: Go follows the camel case naming convention for constants, so I suggest renaming DATABACKUP_LOCK_NAME to DatabackupLockName.

File: pkg/dataload/constants.go (hunk 1 above)
Comment (xliuqq) [reply pkg/dataload/constants.go:20]: fixed.

File: pkg/databackup/constants.go (hunk 2 above)
Comment (xliuqq) [reply pkg/databackup/constants.go]: fixed.

**Comment by cheyang** [comment]
@xliuqq Could you please fix the conflict, thanks.

**Comment by xliuqq** [comment]
> @xliuqq Could you please fix the conflict, thanks.


fixed conflicts.

**Pull request merged**